pkgs/top-level/all-packages.nix

```diff
@@ -1886,6 +1886,8 @@ in
 
   massren = callPackage ../tools/misc/massren { };
 
+  mastodon = callPackage ../servers/mastodon { };
```

Comment:
I recently realized that `all-packages.nix` is supposed to be sorted by directory.  Obviously lots of other people have missed that too. But to avoid adding to the disorder, Mastodon's proper place in the Servers section is down around line 15145, just before `mattermost`.

Reply:
Thanks, I have moved it to the appropriate section